src/main/java/core/basesyntax/strategy/impl/HandlerStrategyImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import static core.basesyntax.Main.OPERATION_HANDLER_MAP;
+
```

Comment:
empty line. And you have import here from you Main class(import static core.basesyntax.Main.OPERATION_HANDLER_MAP;)  (but Main class is just entry point to run your code locally (it's for you)